config/zones.json

```diff
@@ -1524,12 +1524,48 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1010,
+      "biomass": 50
+    },
     "parsers": {
       "production": "US_NY.fetch_production"
     },
     "flag_file_name": "us.png",
     "contributors": [
       "https://github.com/systemcatch"
+      "https://github.com/hanshyde"
+    ]
+  },
+  "US-PJM": {
+    "capacity": {
+      "biomass": null,
+      "coal": 65205,
+      "gas": 63903,
+      "geothermal": null,
+      "hydro": 8675,
+      "hydro storage": null,
+      "nuclear": 33164,
+      "oil": 9148,
+      "solar": null,
+      "wind": 1114,
+      "battery discharge": null,
+      "other": 9112,
+      "unknown": 700,
```

Comment:
Invalid keys that should be removed / merged:
- battery discharge
- other (should be merged with unknown)

Also, values that are 0 should be set to 0 instead of null. Null means that the value is unknown.
This could apply to geothermal / solar / hydro storage / biomass.

Reply:
Copy, the capacity value is not unknown, it is just not being distinctly reported by PJM - solar & biomass to the best of my understanding is reported under PJM's "other renewables" 

"battery discharge" & "other" are in the co2eq_parameters.js  - my bad for including.

This is what I'm attempting to capture/merge between EM & PJM 

![pjm_elelctricity map - merge cat realtime data](https://user-images.githubusercontent.com/33690988/34340088-bb0a98c4-e97d-11e7-8d4b-c51d0c5eec82.png)

Reply:
Couple of formatting issues that are making the tests fail, line 1543 should have a comma between the 2 contributor names, line 1559 does not need a comma at the end.